cmd/dm-ctl/main.go

```diff
@@ -119,22 +119,15 @@ func main() {
 		}
 		os.Exit(0)
 	default:
-		// NOTE:  when `--help` in cmdArgs we need to print out the help msg.)
-		var needPrintHelp bool
+		// NOTE: when `--help` in cmdArgs we need to print out the help msg.
```

Comment:
currently we will print help message for subcommand, so this checking is too loose

```shell
$ bin/dmctl.test DEVEL --master-addr=127.0.0.1:8261 show-ddl-locks --help

Shows un-resolved DDL locks.

Usage:
  dmctl show-ddl-locks [-s source ...] [task-name | task-file] [flags]

Flags:
  -h, --help   help for show-ddl-locks

Global Flags:
  -s, --source strings   MySQL Source ID.
```